website/content/docs/connect/cluster-peering/create-manage-peering.mdx

```diff
@@ -0,0 +1,138 @@
+
+When you connect server agents through cluster peering, they will peer through their default partitions. To connect other partitions through server agents, you must specify the partitions you want to peer using the `partition` parameter.
+
+## Connect service endpoints
+
+After you establish a connection between the clusters, you need to create a configuration entry that defines the services that are available for other clusters. Consul uses this configuration entry to make services and support queries for service mesh available to peered clusters.
+
+First, create a configuration entry and specify the `Kind` as `“exported-services”`.
+
+<CodeBlockConfig filename="peering-config.hcl">
+
+```shell-session
+Kind = "exported-services"
+Partition = "partition-name"
+
+Services = [
+  {
+    ## The name and namespace of the service to export.
+    Name      = "service-name"
+    Namespace = "default"
+
+    ## The list of peer clusters to export the service to.
+    Consumers = [
+      {
+        ## The peer name to reference in config is the one set
+        ## during the peering process.
+        Peer = "cluster-02"
+      }
+    ]
+```
+
+</CodeBlockConfig>
+
+Then, add the configuration entry to your cluster.
+
+```shell-session
+$ consul config write peering-config.hcl
+```
+
+Before you proceed, wait for the clusters to sync and make services available to their peers
+
+## Authorize connections from peers
+
+Before you can call services from peered clusters, you must set service intentions that authorize those clusters to use specific services. Consul prevents services from being exported to unauthorized clusters.
+
+First, create a configuration entry and specify the `Kind` as `“service-intentions”`. Declare the service on "cluster-02" that can access the service in "cluster-01." The following example sets service intentions so that "backend-service" can access "frontend-service."
+
+<CodeBlockConfig filename="peering-intentions.hcl">
+
+```shell-session
+Kind      = "service-intentions"
+Name      = "frontend-service"
+Partition = "partition-name"
+
+Sources = [
+  {
+    Name   = "backend-service" 
+    Peer   = "cluster-02"
+    Action = "allow"
+  }
+]
+```
+
+</CodeBlockConfig>
+
+**Tip:** If the peer’s name is not specified in `Peer`, then Consul assumes that the service is in the local cluster.
+
+Then, add the configuration entry to your cluster.
+
+```shell-session
+$ consul config write peering-intentions.hcl
+```
+
+## Check peered cluster status
+
+To confirm that you peered your clusters, you can [query the `/health/service` endpoint](https://www.consul.io/api-docs/health) of one cluster from the other cluster. For example, in "cluster-02," query the endpoint and add the `peer=cluster-01` parameter to the end of the URL.
+
+```shell-session
+$ curl \
+    http://127.0.0.1:8500/v1/health/service/service-name?peer=cluster-01
+```
+
+Successful connections include `"Status": "passing"` in the output.
```

Comment:
The health status of a service is independent from the status of the peering connection. Verifying that the peering is established should be done by checking whether the services were imported, not that they're in a particular status:
```suggestion
Successful connections include service definitions in the output.
```

For example, if you peer with another cluster that exported a service that is down for some reason. If the service is received after the peering establishment that's what we're looking for. The service being down and not in a "passing" state is not a concern for the peering establishment.